packages/types/lib/driver.ts

```diff
@@ -446,6 +446,8 @@ export interface ExternalDriver<C extends Constraints = BaseDriverCapConstraints
   setAlertText?(text: string): Promise<void>;
   getScreenshot?(): Promise<string>;
   getElementScreenshot?(elementId: string): Promise<string>;
+  getComputedRole?(elementId: string): Promise<string | null>;
+  getComputedLabel?(elementId: string): Promise<string | null>;
```

Comment:
```suggestion
  getComputedRole?(elementId: string): Promise<string?>;
  getComputedLabel?(elementId: string): Promise<string?>;
```

Reply:
oo, nice. i just copied them from another place

Reply:
the ? syntax got an error